index.html

```diff
@@ -30,21 +30,21 @@
 
       <div class="FadeImage"><img src="./view-of-drosia.jpg" alt="View of Drosia" class="Center"/></div>
 
-      
-      <div class ="middle"><iframe width="560" height="315" src="https://www.youtube.com/embed/IZNSk3OnKaI?si=JOLS9Rgq8LYF_vKf" 
-        title="YouTube video player" frameborder="0" 
-        allow="accelerometer; autoplay; clipboard-write; encrypted-media; 
-        gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" 
+
+      <div class ="middle"><iframe width="560" height="315" src="https://www.youtube.com/embed/IZNSk3OnKaI?si=JOLS9Rgq8LYF_vKf"
+        title="YouTube video player" frameborder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media;
+        gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin"
         allowfullscreen></iframe></div>
 
 
-      <iframe style="border-radius:12px" src="https://open.spotify.com/embed/track/4tb7XVRCUzPKeNyhFsZotg?utm_source=generator" 
-      width="25%" height="152" frameBorder="0" allowfullscreen="" 
+      <iframe style="border-radius:12px" src="https://open.spotify.com/embed/track/4tb7XVRCUzPKeNyhFsZotg?utm_source=generator"
+      width="25%" height="152" frameBorder="0" allowfullscreen=""
       allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture" loading="lazy"></iframe>
 
-      <iframe align="right" style="border-radius:12px"  src="https://open.spotify.com/embed/track/2uh88Sbc5kFNV8p2cfrFsP?utm_source=generator" 
+      <iframe align="right" style="border-radius:12px"  src="https://open.spotify.com/embed/track/2uh88Sbc5kFNV8p2cfrFsP?utm_source=generator"
       width="25%" height="152" frameBorder="0" allowfullscreen="" allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture" loading="lazy"> </iframe>
 
-      <script src="thing.js"></script>
+      <script src="index.js"></script>
```

Comment:
I see you renamed thing.js to index which is good for making things more organized. I just named it thing.js cause i kind of treated it like a test file lol. But naming it index is fine to.

Reply:
index.html

CSS file for that file

index.css

JavaScript file for that file...

index.js

> Follow similar patterns and conventions. Really helps keep your code organized and clean